build/testIntegration.sh

```diff
@@ -51,6 +51,22 @@ fi
 
 echo
 
+if [ -n "$2" ]
+then
+    echo
+    echo "Setting environment variable 'ORYX_TEST_REGISTRY_NAME' to provided value '$2'."
+    export ORYX_TEST_REGISTRY_NAME="$2"
+fi
+
+if [ -n "$3" ]
```

Comment:
Would we need this argument? `ORYX_TEST_TAG_SUFFIX` can always depend on the `RELEASE_TAG_NAME` environment variable's value

Reply:
I think in our current cases, we would want to use `RELEASE_TAG_NAME` as the suffix for the tests, but I don't think it hurts to check for this argument being provided in the off-chance that we want to run this script without any tag suffix. Let me know what you think -- I don't mind always setting the tag suffix to `RELEASE_TAG_NAME`